src/rtos/rtos-chibios.ts

```diff
@@ -41,56 +43,136 @@ enum chThreadState {
+    width: 2,
+    headerRow1: '',
+    headerRow2: 'Flags',
+    colGapAfter: 1
+};
+threadTableItems[threadDisplayFields[threadDisplayFields.REFS]] = {
+    width: 2,
+    headerRow1: '',
+    headerRow2: 'Refs',
+    colType: colNumType
+};
+threadTableItems[threadDisplayFields[threadDisplayFields.TIME]] = {
+    width: 2,
+    headerRow1: '',
+    headerRow2: 'Time',
+    colType: colNumType
+};
+threadTableItems[threadDisplayFields[threadDisplayFields.WTOBJP]] = {
     width: 4,
     headerRow1: 'Wait',
     headerRow2: 'Obj/Msg',
     colGapBefore: 1
 };
-ChibiOSItems[DisplayFields[DisplayFields.STATS_N]] = {
+threadTableItems[threadDisplayFields[threadDisplayFields.STATS_N]] = {
     width: 4,
     headerRow1: 'Stats',
     headerRow2: 'Switches',
     colType: colNumType
 };
-ChibiOSItems[DisplayFields[DisplayFields.STATS_WORST]] = {
+threadTableItems[threadDisplayFields[threadDisplayFields.STATS_WORST]] = {
     width: 4,
     headerRow1: '',
     headerRow2: 'Worst Path',
     colType: colNumType
 };
-ChibiOSItems[DisplayFields[DisplayFields.STATS_CUMULATIVE]] = {
+threadTableItems[threadDisplayFields[threadDisplayFields.STATS_CUMULATIVE]] = {
     width: 4,
     headerRow1: '',
     headerRow2: 'Cumulative Time',
     colType: colNumType
 };
-ChibiOSItems[DisplayFields[DisplayFields.STACK_CURRENT_USAGE]] = {
+threadTableItems[threadDisplayFields[threadDisplayFields.STACK_CURRENT_USAGE]] = {
     width: 3,
     headerRow1: 'Stack',
     headerRow2: '',
     colType: colNumType
 };
-ChibiOSItems[DisplayFields[DisplayFields.STACK_PEAK_USAGE]] = {
+threadTableItems[threadDisplayFields[threadDisplayFields.STACK_PEAK_USAGE]] = {
     width: 3,
     headerRow1: '',
     headerRow2: '',
     colType: colNumType
 };
 
-const DisplayFieldNames: string[] = Object.keys(ChibiOSItems);
+const threadDisplayFieldNames: string[] = Object.keys(threadTableItems);
+
+const globalInfoCols = [{ columnDataKey: 'name', title: 'Name' }, { columnDataKey: 'value', title: 'Value' } ];
+
+const virtualTimersCols = [{ columnDataKey: 'timer', title: 'Timer' },
+                           { columnDataKey: 'time', title: 'Time' },
+                           { columnDataKey: 'delta', title: 'Delta' },
+                           { columnDataKey: 'callback', title: 'Callback' },
+                           { columnDataKey: 'params', title: 'Parameters' },
+                           { columnDataKey: 'last', title: 'Last Deadline' },
+                           { columnDataKey: 'reload', title: 'Reload' }];
+
+const statisticsCols = [{ columnDataKey: 'description', title: 'Measured Section' },
+                        { columnDataKey: 'best', title: 'Best Case' },
+                        { columnDataKey: 'worst', title: 'Worst Case' },
+                        { columnDataKey: 'counter', title: 'Iterations' },
+                        { columnDataKey: 'cumulative', title: 'Cumulative Time' } ];
+
+const traceCols = [{ columnDataKey: 'event', title: 'Event' },
```

Comment:
Is it intended that `traceCols` is not used?

Reply:
yes, trace code is not in use now. It takes to long to get trace data.

Reply:
👍